src/ray/core_worker/core_worker.cc

```diff
@@ -792,32 +786,24 @@ Status CoreWorker::CreateActor(const RayFunction &function,
                       worker_context_.GetCurrentTaskID(), next_task_index, GetCallerId(),
                       rpc_address_, function, args, 1, actor_creation_options.resources,
                       actor_creation_options.placement_resources,
-                      actor_creation_options.is_direct_call ? TaskTransportType::DIRECT
-                                                            : TaskTransportType::RAYLET,
-                      &return_ids);
+                      TaskTransportType::DIRECT, &return_ids);
   builder.SetActorCreationTaskSpec(
       actor_id, actor_creation_options.max_reconstructions,
       actor_creation_options.dynamic_worker_options,
-      actor_creation_options.is_direct_call, actor_creation_options.max_concurrency,
+      /*is_direct_call=*/true, actor_creation_options.max_concurrency,
```

Comment:
Can we remove this too?

Reply:
We can but want to minimize the surface area of these changes because I've run into unexpected issues in the past when trying to remove these. Will remove from the task spec entirely in a future PR.